lib/full-site-editing/templates-utils.php

```diff
@@ -59,63 +54,23 @@ function gutenberg_render_templates_lists_custom_column( $column_name, $post_id
 		}
 		$themes        = array();
 		$is_file_based = false;
+		$is_original   = false;
 		foreach ( $terms as $term ) {
 			if ( '_wp_file_based' === $term->slug ) {
 				$is_file_based = true;
+			} else if ( '_wp_is_original' === $term->slug ) {
+				$is_original = true;
 			} else {
 				$themes[] = esc_html( wp_get_theme( $term->slug ) );
 			}
 		}
```

Comment:
Wonder if we should extract this as a util function. Reusing it in `filter_rest_prepare_add_wp_theme_terms`